### PR TITLE
Expected pcrs

### PR DESCRIPTION
**Pull request by krystian-hebel**
Added ability to print expected PCR values in order to easier catch any potential issues.

### CODE REVIEW

**Comment by krystian-hebel** [comment]
Not tested in this exact form yet, this was in big part manually rewritten because copying files between Qubes dom0 and development computer is hard.

**Comment by krystian-hebel** [comment]
After fixing file permissions and some typos it works as expected for TPM2.0.

File: sbin/txt-tpm1-evt-log-parser.awk [hunk 1]
```diff
@@ -69,9 +71,14 @@ BEGIN {
 		printf("    Digests:\n")
 		printf("      SHA1: ")
 		hexdump($3, 20)
+		sym = "SHA1_" x2n($1, 4)
+		SYMTAB[sym] = SYMTAB[sym] hex_noprint($4, a[2]) "\n"
```
Comment (SergiiDmytruk) [inline sbin/txt-tpm1-evt-log-parser.awk]:
```suggestion
		SYMTAB[sym] = SYMTAB[sym] hex_noprint($3, 20) "\n"
```

**Review by SergiiDmytruk — CHANGES_REQUESTED** [review]
Will need to look closer, but TPM1 script has an issue for sure.

File: sbin/txt-tpm1-evt-log-parser.awk (hunk 1 above)
Comment (krystian-hebel) [reply sbin/txt-tpm1-evt-log-parser.awk]: Right, copy-pasted and forgot to check it. Fixed in https://github.com/TrenchBoot/qubes-antievilmaid/compare/a1196c77efb591cee69ddcce4df20c384a6ffca9..2cda9788812bcdb2531f3314f705a916997c71ae

**Review by SergiiDmytruk — APPROVED** [review]
Values of PCR17 don't match on KGPE-16, but it's not the script's fault, it computes everything correctly.  Maybe some package is missing one of the fixes.

**Pull request merged**